src/Microsoft.DocAsCode.Dfm/Rules/DfmFencesBlockRule.cs

```diff
@@ -90,7 +90,7 @@ private IMarkdownToken TryMatchOld(IMarkdownParser parser, IMarkdownParsingConte
             var title = match.Groups["title"]?.Value;
             var pathQueryOption = ParsePathQueryString(match.Groups["option"]?.Value, match.Groups["optionValue"]?.Value);
 
-            return new DfmFencesBlockToken(this, parser.Context, name, path, sourceInfo, lang, title, pathQueryOption);
+            return new DfmFencesBlockToken(this, parser.Context, name, path, sourceInfo, lang, title, pathQueryOption, pathQueryOption == null ? match.Groups["option"].Value + match.Groups["optionValue"].Value : null);
```

Comment:
it would be better even if those value must exist since pathQueryOption is not null.

---
In reply to: [134927052](https://github.com/dotnet/docfx/pull/1996#discussion_r134927052) [](ancestors = 134927052)